pkgs/servers/sql/mariadb/default.nix

```diff
@@ -184,13 +183,12 @@ server = stdenv.mkDerivation (common // {
     "-DWITHOUT_ROCKSDB=1"
   ] ++ optional (!stdenv.hostPlatform.isDarwin && withStorageRocks) [
     "-DWITH_ROCKSDB_JEMALLOC=ON"
-  ] ++ optional (stdenv.hostPlatform.isDarwin || stdenv.hostPlatform.isMusl || !withStorageToku) [
-    "-DWITHOUT_TOKUDB=1"
-  ] ++ optional (!stdenv.hostPlatform.isDarwin && withStorageToku) [
-    "-DWITH_JEMALLOC=static"
-  ] ++ optional stdenv.hostPlatform.isDarwin [
+  ] ++ optional (!stdenv.hostPlatform.isDarwin) [
+    "-DWITH_JEMALLOC=yes"
```

Comment:
Why are we switching from jemalloc=static? Why were we doing that in the first place?

Reply:
This option forces cmake to build with jemalloc support:
https://github.com/MariaDB/server/blob/10.5/cmake/jemalloc.cmake#L11-L12
This is not static. Static build with option `"-DWITH_JEMALLOC=static"`
https://github.com/MariaDB/server/blob/10.5/cmake/jemalloc.cmake#L18-L21

Reply:
Option "-DWITH_JEMALLOC=static" was needed to correct build TokuDB plugin.